.travis.yml

```diff
@@ -1,6 +1,7 @@
 env:
  global:
   - LIBRDKAFKA_VERSION=v1.9.2
+  - LIBRDKAFKA_SRC_CODE_VERSION=master
```

Comment:
tiny nit: I'd drop the "_CODE", a bit redundant.

Reply:
Done